.circleci/config.yml

```diff
@@ -301,7 +301,7 @@ jobs:
               -wait tcp://${HOSTNAME}:3306 \
               -wait tcp://${HOSTNAME}:9200 \
               -timeout 60s \
-                ~/.local/bin/pytest
+                ~/.local/bin/pytest --disable-warnings
```

Comment:
Not sure it's a good idea to hide them. We need to be aware of those warnings, it's not an issue that could make the CI fail, so...

Reply:
That's why I do not use this option to run tests locally. But about CI, I think to remove these warnings help you to identify quickly what's wrong.

Reply:
+1 to remove. The warnings associated with our code have been solved and the remaining warnings are raised by our dependencies so there's nothing we can do about them.

Reply:
I do agree, that this is not our responsibility to fix it, but it's not worth being blind about it. :woman_shrugging:

Reply:
And, this globally removes warning for which we could be responsible of in future developments.

Reply:
If you persist in your idea to remove those warnings, you can filter them instead: https://docs.pytest.org/en/stable/warnings.html

Reply:
I can use this settings that ignores and **really ignores** all warnings coming from another packages than `richie` or `tests`.
```
filterwarnings = 
    ignore:::(?!(tests|richie))
```

On the other hand by using `--disable-warnings`, pytest does not ignore warnings, it does not display details of them in the stacktrace. But If you read the resume you see the number of warnings catched. And locally you can run `make test-back` and see all details.

Reply:
We need a consensus about this question. I created a new Pull request (#1186) for this aspect.